### DISCUSSION TITLE
ISO 2.3.2 image after installation without so-zeek

### DISCUSSION THREAD

**Discussion by ldceven** [Security-Onion-Solutions/securityonion#1925] [Unsupported Versions]
Hello 
 I installed the ISO SO 2.3.2 image. After installation, other services are normal, but I found that there is no so-zeek service. Is this phenomenon normal? Can't SO 2.3 run zeek and Suricata at the same time?
Do I need to install so-zeek separately?

Thank you very much.

[so_2020-11-16_17-34-52.log](https://github.com/Security-Onion-Solutions/securityonion/files/5545416/so_2020-11-16_17-34-52.log)

**Comment by weslambert** [comment]
Did you choose Suricata for metadata?  Is there a Zeek Docker image on the box?

`docker images | grep zeek`

Are there any clues if you run `docker logs -f so-zeek`
